THIRD-PARTY-LICENSES.txt

```diff
@@ -148,7 +148,7 @@ Lists of 360 third-party dependencies.
      (MIT) ficus (com.iheart:ficus_2.12:1.4.4 - http://iheartradio.github.io/ficus)
      (The Apache Software License, Version 2.0) FindBugs-jsr305 (com.google.code.findbugs:jsr305:3.0.2 - http://findbugs.sourceforge.net/)
      (The Apache Software License, Version 2.0) GeAnTyRef (io.leangen.geantyref:geantyref:1.3.7 - https://github.com/leangen/geantyref)
-     (The MIT license) GitHub API for Java (org.kohsuke:github-api:1.101 - https://github-api.kohsuke.org/)
+     (The MIT license) GitHub API for Java (org.kohsuke:github-api:1.116 - https://github-api.kohsuke.org/)
```

Comment:
This looks like it occurred in a different PR, please create a ticket to make a CI build fail if this file is changing but not checked in.

Reply:
Done. Created a ticket to make a CI build fail.